internal/slice/slice.go

```diff
@@ -0,0 +1,37 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package slice
```

Comment:
is this package used anywhere in the current code? I think this might have been only in use for the finalizers slice?

Reply:
nice find, doesn't look like it is at the moment, I'll trash and we can add back later on.